### ISSUE TITLE
Folder Support

### ISSUE THREAD

**Issue by TCROC** [jeffjadulco/unity-guid-regenerator#1]
It would be really nice to have folder support so we can just click on the folder containing the assets we want to update.  I implemented a little hack with [AssetDatabase.FindAssets](https://docs.unity3d.com/ScriptReference/AssetDatabase.FindAssets.html).  It caused it to fall into an endless loop, but when using the Task Manager and ending the Unity Task, I was happy to find that the guids were updated :).

TL; DR;

Feature request for folder support.

Edit: Typo
